modules/transactions/src/main/java/org/apache/ignite/internal/tx/views/TransactionsViewProvider.java

```diff
@@ -93,27 +86,20 @@ public SystemView<?> get() {
     static class TxInfoDataSource implements Iterable<TxInfo> {
         private final UUID localNodeId;
 
-        private final Iterable<UUID> roTxIds;
-
         private final Map<UUID, TxStateMeta> rwTxStates;
```

Comment:
if `rwTxStates` now keeps states for both RO and RW transactions, it's better to rename it accordingly